src/main/java/com/booleanuk/core/Exercise.java

```diff
@@ -40,74 +40,86 @@ public String sayGoodMorning(boolean isMorning) {
     // 1. What will the returned value be if I run sayGoodMorning(false)?
     // Change the returned value in the method below to your answer. It is case-sensitive.
     public String one() {
-        return "";
+        return "Good day!";
     }
 
     // 2. What will the output be if I run sayGoodMorning(true)?
     // Change the returned value in the method below to your answer. It is case-sensitive.
     public String two() {
-        return "";
+        return "Good morning!";
     }
 
+
     // 3. What will the output be if I run sayGoodMorning("Hello".equals("Hello"))?
     // Change the returned value in the method below to your answer. It is case-sensitive.
     public String three() {
-        return "";
+        return "Good morning!";
     }
 
     // 4. What will the output be if I run sayGoodMorning(!"A word".equals("Another word"))
     public String four() {
-        return "";
+        return "Good morning!";
     }
 
     // 5. What will the output be if I run sayGoodMorning(25 != 25)
     public String five() {
-        return "";
+        return "Good day!";
     }
 
     // 6. Use a conditional statement to return "Correct!" if the input is more than 7
     // or "Wrong!" if not
     public String six(int num) {
-        return "Not implemented yet!";
+
+
+        return num > 7 ? "Correct!" : "Wrong!";
     }
 
     // 7. Use a conditional statement to return "Correct!" if the input is false
     // or "Wrong!" if not
     public String seven(boolean bool) {
-        return "Not implemented yet!";
+        return bool ? "Wrong!" : "Correct!";
     }
 
     // 8. Use a conditional statement to return "Correct!" if numOne is more than or equal to numTwo
     // or "Wrong!" if not
     public String eight(int numOne, int numTwo) {
-        return "Not implemented yet!";
+        return numOne >= numTwo ? "Correct!" : "Wrong!";
     }
 
     // 9. Use a conditional statement to return true if the array provided is not empty
     // or false if it is empty
     public boolean nine(int[] nums) {
-        return false;
+        return nums.length > 0;
     }
 
     // 10. Use a conditional statement to return true if the provided string contains the word
     // "milk", or false if not
     // https://www.w3schools.com/java/java_ref_string.asp
     public boolean ten(String sentence) {
-        return false;
+        return sentence.contains("milk");
```

Comment:
This will return the correct answer, but is not technically a conditional statement (as per instructions).

Reply:
Would it be technically correct if i wrote:

If(sentence.contains("milk") {
return true;
}

Im just used to skipping the if statement if i can write it in the return.

Or am i thinking wrongly?

Reply:
What you had written before is perfectly acceptable, and likely the way you might do that check in "_real life_".  
However, specifically for this task it asks to use a conditional statement, so yes, your new suggestion is doing that.

You could also write it in another way, using a [`Ternary`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_operator). (check the link for more info)

So your `if` statement as a `Ternary` would be:

`return sentence.contains("milk") ? true : false;`

You are correct in thinking it adds unnecessary complexity here. The idea is to expose you to concepts. 

For example, using `Ternary` or `If` gives you control over the output, for example, you might want to do something like:

`return sentence.contains("milk") ? "got milk": "no milk";`